flutter/lib/src/default_integrations.dart

```diff
@@ -99,7 +99,9 @@ class FlutterErrorIntegration extends Integration<SentryFlutterOptions> {
               : null,
         );
 
-        await hub.captureEvent(event, stackTrace: errorDetails.stack);
+        await hub.captureEvent(event,
+            stackTrace: errorDetails.stack,
+            hint: Hint.fromMap({'errorDetails': errorDetails}));
```

Comment:
For cases like this, the key should be `syntheticException`.
See `SentryAppender` on Java.